doc/client_lua_api.md

```diff
@@ -754,6 +759,20 @@ Call these functions only at load time!
     * returns reference to mod private `StorageRef`
     * must be called during mod load time
 
+### Mod channels
+* `minetest.mod_channel_join(channel_name)`
+	* Client joins channel `channel_name`. If channel has no member it's created. You 
+	  should listen from incoming messages with `minetest.register_on_modchannel_message`
+	  call to receive incoming messages. Warning, this function is asynchronous,
+	  you should use a minetest.after(2, function() ... end) at least, to permit server
+	  acknowledge your request
+* `minetest.mod_channel_leave(channel_name)`
+	* Client leaves channel `channel_name`. No more incoming or outgoing messages can be
+	  sent to this channel from client mods.
+* `minetest.mod_channel_send_msg(channel_name, message)`
```

Comment:
just `mod_channel_send` would be more logical, as we also have `chat_player_send` (instead of `chat_player_send_msg`)

Reply:
okay :)